kedro-datasets/kedro_datasets/partitions/incremental_dataset.py

```diff
@@ -40,16 +40,9 @@ class IncrementalDataset(PartitionedDataset):
 
         >>> from kedro_datasets.partitions import IncrementalDataset
         >>>
-        >>> # these credentials will be passed to:
-        >>> # a) 'fsspec.filesystem()' call,
-        >>> # b) the dataset initializer,
-        >>> # c) the checkpoint initializer
-        >>> credentials = {"key1": "secret1", "key2": "secret2"}
-        >>>
         >>> dataset = IncrementalDataset(
-        ...     path="s3://bucket-name/path/to/folder",
+        ...     path="path/to/folder",
```

Comment:
How did this work with the dummy path? 🤔

Reply:
`fsspec` will create a path. However, we should use use `tmp_path` fixture in order to avoid polluting the env.